app/javascript/src/components/ItemList/ItemListItem.jsx

```diff
@@ -0,0 +1,85 @@
+// Item list item
+import { h } from 'preact';
+import { PropTypes } from 'preact-compat';
+
+export const ItemListItem = ({ item, children }) => {
+  const adaptedItem = {
```

Comment:
this ugly trick is due to the fact that even though 90% of the functionality is the same, the indexed object is slightly different, so I use an "adapter" to get what I need